SPARQLWrapper/SmartWrapper.py

```diff
@@ -212,41 +211,57 @@ def __getitem__(self, key):
         :rtype: array of variable -> :class:`Value`  dictionaries
         """
 
-        def _checkKeys(keys):
+        def _checkKeys(keys: Union[List[Any], Tuple[Any, ...]]) -> bool:
             if len(keys) == 0:
                 return False
             for k in keys:
-                if not isinstance(k, str) or k not in self.variables:
+                if (
```

Comment:
Looks like this can go into a separate PR.

Reply:
934e334